src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -1789,6 +1801,7 @@ public partial class DataAdapter : System.ComponentModel.Component, System.Data.
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public virtual bool ShouldSerializeFillLoadOption() { throw null; }
         protected virtual bool ShouldSerializeTableMappings() { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("DataTypes of schema table cannot be statically analyzed.")]
```

Comment:
This message seems a bit off for the method. It isn't obvious what "schema table" is here.

Reply:
I've changed it and hope it's clearer now